Component-Page/component-page.css

```diff
@@ -0,0 +1,48 @@
+h1.sub-heading {
+    font-size: 1.5rem;
+    margin: 1rem 0;
+}
+
+.demo-with-code {
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    align-items: center;
+    margin-bottom: 3rem;
+}
+
+.demo-div {
+    display: flex;
+    flex-wrap: wrap;
+    background-color: #F7E800;
+    justify-content: center;
+    gap: 1.5rem;
+    align-items: center;
+    color: black;
+    border-radius: 5px;
+    padding: 20px;
```

Comment:
Padding more than 10px can be given as rem.

Reply:
Noted. I've updated padding in rem units wherever suitable. for eg. if I have a padding for an element as 10px 12px (top and sides), I'd rather have it as px than convert it into a decimal rem.